modules/blocks.js

```diff
@@ -54,25 +54,25 @@ __private.isActive = false;
 class Blocks {
 	constructor(cb, scope) {
 		library = {
-			logger: scope.logger,
+			logger: scope.logger.child({ module: 'blocks' }),
```

Comment:
More nicer would be to have:

```
modules/blocks
modules/blocks/api
modules/blocks/verify 
```
And so one.

Reply:
since we have `module` key in the log, wouldn't it be redundant?